core/src/templates/components/media/media.twig

```diff
@@ -30,7 +30,7 @@
 #}
 
 {# Set link to null if there is no content to prevent an empty link #}
-{%- if media_link is not empty and media_image_src is empty and media_audio_src is empty and media_video_src is empty -%}
+{%- if media_image_src is empty and media_audio_src is empty and media_video_src is empty -%}
```

Comment:
The media link doesn't have to be tied to the media source. I would almost go the other way and remove all the src checks.

Reply:
What I'd like to happen is that the link does not appear if there is no image. Right now, the link will appear even if it is the only bit of content provided. My thought was that I'd look to see if the media source was empty and make the link empty. Would it make sense instead to have all the fields as required?

Reply:
I'd rather have these components have less logic in them and use explicitly set variables. If this template gets passed a variable, then it should use it. 

So if you want to avoid having a link when there is no image, you would not pass in a media_link variable.

Reply:
Does Drupal or something else make that difficult?

Reply:
I hadn't thought about it that way before. Does it make sense to you to have the media a required field?

Reply:
If a client wanted a link or a caption without an image, they'd use a different paragraph type. If there's a link, there needs to be an image to link, right?

Reply:
Ok. We talked more in the standup and I see the way. I'll revert what I did. :)